src/abstractitensornetwork.jl

```diff
@@ -584,37 +585,49 @@ function LinearAlgebra.factorize(tn::AbstractITensorNetwork, edge::Pair; kwargs.
 end
 
 # For ambiguity error; TODO: decide whether to use graph mutating methods when resulting graph is unchanged?
-function _orthogonalize_edge(tn::AbstractITensorNetwork, edge::AbstractEdge; kwargs...)
+function orthogonalize_path(tn::AbstractITensorNetwork, edge::AbstractEdge; kwargs...)
+  return orthogonalize_path(tn, [edge]; kwargs...)
+end
+
+function orthogonalize_path(tn::AbstractITensorNetwork, edge::Pair; kwargs...)
+  return orthogonalize_path(tn, edgetype(tn)(edge); kwargs...)
+end
+
+# For ambiguity error; TODO: decide whether to use graph mutating methods when resulting graph is unchanged?
+function orthogonalize_path(
```

Comment:
I would prefer calling this `orthogonalize_walk` since the set of edges may in general define a walk rather than a path (https://en.wikipedia.org/wiki/Path_(graph_theory)#Walk,_trail,_and_path).